PopPUNK/visualise.py

```diff
@@ -268,36 +295,67 @@ def main():
                                                return_dict = True)
     # Join clusters with query clusters if required
     if not self:
-        if args.previous_query_clustering is not None:
-            prev_query_clustering = args.previous_query_clustering
+        if previous_query_clustering is not None:
+            prev_query_clustering = previous_query_clustering
         else:
-            prev_query_clustering = args.query_db
+            prev_query_clustering = query_db
 
         queryIsolateClustering = readIsolateTypeFromCsv(
-                prev_query_clustering + '/' + os.path.basename(prev_query_clustering) + suffix,
+                prev_query_clustering + suffix,
```

Comment:
Is this change correct? What if `--previous-query-clustering` was something like `../db_assign/db_assign_clusters.csv`